### PR TITLE
Created and updated

### PR DESCRIPTION
**Pull request by huyluong22**

### CODE REVIEW

File: scripts/add-fields-for-users.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+CREATE TABLE users (
+    id char(36) PRIMARY KEY,
+    user_id VARCHAR(255) NOT NULL,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    sex VARCHAR(50),
+    email VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```
Comment (SonNgocTruong) [inline scripts/add-fields-for-users.sql:11]: I do not want to edit the existing SQL file.
The purpose is to create a new file and modify table to add new columns using `ALTER TABLE`.
Example: https://www.w3schools.com/sql/sql_alter.asp

File: userservice/src/main/java/com/infodation/userservice/controllers/UsersController.java [hunk 2]
```diff
@@ -98,7 +98,7 @@ public ResponseEntity<ApiResponse<?>> delete(@PathVariable("userId") String user
         } else {
             userService.delete(userId);
             status = HttpStatus.OK;
-            message =  String.format("User with ID %s not found", userId);
+            message = String.format("User with ID %s deleted successfully", userId);
```
Comment (SonNgocTruong) [inline userservice/src/main/java/com/infodation/userservice/controllers/UsersController.java:101]: Good point

**Comment by SonNgocTruong** [comment]
Hi @huyluong22, please help add ticket proof (text and images into your PR for reviewer to check)

**Comment by huynhatluong22** [comment]
image
image

**Pull request merged**